.github/dependabot.yml

```diff
@@ -0,0 +1,58 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Standalone deps
+      - dependency-name: "mongodb"
+      - dependency-name: "bson"
+      # All the mongosh deps
+      - dependency-name: "@mongosh/node-runtime-worker-thread"
+      - dependency-name: "@mongosh/browser-repl"
+      - dependency-name: "@mongosh/logging"
```

Comment:
This should probably include `@mongodb-js/oidc-plugin` and `@mongodb-js/devtools-connect` as well, when those update we typically do bumps for mongosh + those packages